packages/all/project.json

```diff
@@ -24,6 +24,12 @@
         "fix": {}
       }
     },
+    "lint:changeDetection": {
+      "executor": "nx:noop",
+      "inputs": [],
+      "outputs": [],
+      "dependsOn": ["ag-charts-community:lint:changeDetection", "ag-charts-enterprise:lint:changeDetection"]
```

Comment:
Nit: Use `^lint:change-detection` to automatically include all projects with a `lint:change-detection` target`